CHANGELOG.md

```diff
@@ -7,7 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes -->
 
-## [v0.91.0-sumo-0]
+## [v0.91.0-sumo-1]
```

Comment:
Is this a bug fix or a feature someone is waiting for that would warrant a `sumo-1` release?

Note that the upstream has already released `v0.92.0` yesterday, so why not just release `v0.92.0-sumo-0` instead or `v0.91.0-sumo-1`?

Reply:
@astencel-sumo I'm not sure how to pull in a new upstream release yet and hadn't thought to check. Is it worth adding that as part of the release documentation?

I'm just heading to bed as it's past midnight but we need a release to go out today with the statically linked linux_arm64 FIPS build. Is it quite involved to pull in a new upstream release and if so is it something that can be done by EOD?

Reply:
Here's documentation on updating to a new version of upstream Otelcol: https://github.com/SumoLogic/sumologic-otel-collector/blob/v0.91.0-sumo-0/docs/release.md#updating-ot-to-next-version

We are working towards releasing `v0.92.0-sumo-0` today.

Reply:
There are some issues that make releasing `v0.92.0-sumo-0` a pain, so let's move forward with releasing `v0.91.0-sumo-1` if that's OK. Sorry for the mix-up!